tools/matchms/matchms_formatter.xml

```diff
@@ -31,8 +31,8 @@
     </configfiles>
 
     <inputs>
-        <param label="Scores Table" name="scores" type="data" format="tsv" help="Scores output table." />
-        <param label="Matches Table" name="matches" type="data" format="tsv" help="Scores output table." />
+        <param label="Scores Table" name="scores" type="data" format="table,csv,tsv" help="Scores output table." />
```

Comment:
I think this is fine but I am not clear on how `table` behaves -- as you mentioned it is not a tabular format in Galaxy so it'd make sense to drop it.

Reply:
I have removed `table` format in 45ab1df.